Translate comments in SpellAuras.cpp from French to English

DeltaF1 opened the pull request:
## 🍰 Pullrequest
Working on translating aura descriptions from french to english

### Issues
- relates [#683](https://github.com/vmangos/core/issues/683)

### Todo / Checklist
- [x] Random code comments
- [x] _IsExclusiveSpellAura buff list comments

DeltaF1 committed:
First pass

- Translate some easy comments. Some like "Onyxia buff" are ambiguous

0blu commented:
Just in case you want to translate even more stuff:
In `*commands.cpp` there are a lot of  `"oui"/"non"` instead of  `"yes" /"no"` .

(_Reminds me of the Futurama gag <https://www.youtube.com/watch?v=pwODwwgE6rA>_ 🤣)

DeltaF1 committed:
Add names from database instead

DeltaF1 committed:
Random comment fixes

DeltaF1 commented:
> In *commands.cpp there are a lot of "oui"/"non" instead of "yes" /"no" .

What does PSendSysMessage do? Are there scripts or something that depend on those values being OUI/NON?

DeltaF1 committed:
More random comments

0blu commented:
`PSendSysMessage` is just a function that sends a message back to the player.
There should be no conflict with any script or other dependency.

DeltaF1 commented:
I know that people do things like write scripts to parse combat log messages which is why I asked. I'll make a separate PR for it just in case.

DeltaF1 committed:
More fixes

- Translate a couple of small comments
- Expand comment about Gobl/Gnomish Rocket Boot probabilities. FIXME:
  Research this on retail

DeltaF1 committed:
Best attempt at last comments

- Google translated a comment that seemed more conversational than
  technical, since I was unsure of its intent
- Expand description of Ghost Wolf/Water Walking interaction

DeltaF1 commented:
@0blu Are you able to review this PR? No rush at all, I just want to make sure it's not in limbo since it started as a draft

ratkosrb commented:
I don't think he speaks French to be able to review it.

0blu commented on src/game/Spells/SpellAuras.cpp:
```diff
@@ -952,6 +952,7 @@ bool Aura::CanProcFrom(SpellEntry const* spell, uint32 EventProcEx, uint32 procE
     uint64 mask = sSpellMgr.GetSpellAffectMask(GetId(), GetEffIndex());
 
     // Nostalrius: c'est la moindre des choses d'utiliser un peu 'spell_proc_event' non ?
```
Can you please replace this too, no need to put the translation below it.
Although I am unsure what this means in this context.
@ratkosrb maybe you can give us a bit of context.

DeltaF1 replied:
Yeah I was confused about what this means. I took it as more of a conversational note rather than explaining what the code was doing, so I wasn't sure whether to translate it at all or just leave it as-is

0blu commented on src/game/Spells/SpellAuras.cpp:
```diff
@@ -4268,7 +4271,7 @@ void Aura::HandleModMechanicImmunityMask(bool apply, bool /*Real*/)
 }
 
-//this method is called whenever we add / remove aura which gives m_target some imunity to some spell effect
+//this method is called whenever we add / remove aura which gives m_target some immunity to some spell effect
```
Adding a space
```suggestion
// this method is called whenever we add / remove aura which gives m_target some immunity to some spell effect
```

0blu reviewed (commented):
Unfortunately I don't speak French, but I used deepL to get a second opinion on the translations.
Other than my addition your changes look good.
Many thanks for your effort!

DeltaF1 committed:
Update src/game/Spells/SpellAuras.cpp

Co-authored-by: _BLU <dot0blu@gmail.com>

Daribon commented on src/game/Spells/SpellAuras.cpp:
```diff
@@ -8430,70 +8433,69 @@ bool _IsExclusiveSpellAura(SpellEntry const* spellproto, SpellEffectIndex eff, A
+        case 18192: // Increased Agility
+        case 18191: // Increased Stamina
+        case 25661: // Increased Stamina
+        case 24427: // Toasty
```
Why was this changed to Toasty? Spell 24427 is Diamond Flask.
There is no spell named Toasty in vanilla.

DeltaF1 replied:
I pulled the data for these comments straight from the vmangos database. It looks like it was called Toasty in one of the older builds. I can try to write a query to only pull the latest names if that's most useful.

| name | build |
|--------|--------|
| Toasty | 4695 |
| Diamond Flask | 4878 | 
| Diamond Flask | 5302 |
